sass/style-short.scss

```diff
@@ -1,5 +1,6 @@
 #logo {
-  height: 100px;
+  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
```

Comment:
The path should be relative to the css (in dist), not the scss

On Mon, Apr 20, 2015 at 9:26 PM sundialmusic <notifications@github.com>
wrote:

> In sass/style-short.scss
> <https://github.com/sundialmusic/gravitycoast/pull/7#discussion_r28749256>
> :
>
> > @@ -1,5 +1,6 @@
> >  #logo {
> > -  height: 100px;
> > +  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
>
> Yeah, I thought that we re-arranged to account for this yesterday, but I
> was finding that the shortened logo image wasn't loading. The logo images
> were in the css/images folder because the logo was being displayed by being
> the background image of a div... so from the css file or sass file, it
> would have to go UP one directory to see the images folder. Child's play
> for you, but how do we work around this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sundialmusic/gravitycoast/pull/7/files#r28749256>.
>